cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -289,6 +290,11 @@ def _add_resources(self):
                 cw_log_group_name=self.log_group.log_group_name if self.config.is_cw_logging_enabled else None,
             )
 
+        # Budgets
+        if self.config.dev_settings and self.config.dev_settings.budgets:
+            cost_budgets = CostBudgets(self, self.config)
+            self.budget_list = cost_budgets.create_budgets()
```

Comment:
Shall we move the create_budgets function into budget_builder.py? 
Add the end of the _init_ function, we can add the resources, as what we do for other builder stack

Reply:
Good call. Done!